collect_app/src/main/res/layout/form_entry_end.xml

```diff
@@ -77,46 +77,48 @@ the specific language governing permissions and limitations under the License.
             </androidx.constraintlayout.widget.ConstraintLayout>
         </com.google.android.material.card.MaterialCardView>
 
-        <androidx.constraintlayout.widget.ConstraintLayout
+        <LinearLayout
             android:id="@+id/buttons_container"
-            android:layout_width="0dp"
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
             android:layout_marginTop="64dp"
-            app:layout_constraintWidth_max="640dp"
             app:layout_constraintBottom_toBottomOf="parent"
             app:layout_constraintEnd_toEndOf="parent"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/form_edits_warning">
+            app:layout_constraintTop_toBottomOf="@+id/form_edits_warning"
+            app:layout_constraintWidth_max="640dp">
```

Comment:
We should create a const for this value since it's already used in another place.